client/src/main/java/org/spine3/base/Commands.java

```diff
@@ -269,7 +270,8 @@ public int compare(Command o1, Command o2) {
      * Checks if the file is for commands.
      *
      * @param file a descriptor of a {@code .proto} file to check
-     * @return {@code true} if the file name ends with the {@link #FILE_NAME_SUFFIX}, {@code false} otherwise
+     * @return {@code true} if the file name ends with the {@link #FILE_NAME_SUFFIX},
+     *                      {@code false} otherwise
```

Comment:
Possible, it's nice formatting for a Javadoc, but `idea`'s `reformat code` will reformat a Javadoc with such formatting. We should avoid this here and in similar cases.